src/components/OnchainIdentity/SocialVerification.tsx

```diff
@@ -230,20 +231,29 @@ const SocialVerification = ({ className, socials, onCancel, startLoading, closeM
 		startLoading({ isLoading: false, message: '' });
 	};
 
+	const handleDeleteSocialVerificationRef = async () => {
```

Comment:
handleDeleteSocialVerificationRef should be called from where `identitySetViaPolkassembly` goes true